src/src/rx_main.cpp

```diff
@@ -546,10 +553,21 @@ void GotConnection()
 void ICACHE_RAM_ATTR ProcessRFPacket()
 {
     beginProcessing = micros();
-    uint8_t calculatedCRC = ota_crc.calc(Radio.RXdataBuffer, 7) + CRCCaesarCipher;
-    uint8_t inCRC = Radio.RXdataBuffer[7];
+
+    if (getParity(Radio.RXdataBuffer, 8))
```

Comment:
Received parity bit must be stored and cleared before calc.

Reply:
I think the idea of setting the parity bit only if `getParity` returns 1 is so that `getParity` on the receiving end should always be 0 if there are no errors. So if `getParity` returns 1 on the receiving side then it's a reception error.

Reply:
Correct, parity should be 0.  And parity bit and CRC is already cleared in the packet before calculating the CRC.

Reply:
True... Sorry, should not review things on phone 😄